docs/faq/code-analysis/which-metrics-does-codacy-calculate.md

```diff
@@ -145,8 +145,8 @@ Once the coverage setup is complete, Codacy displays coverage data on the follow
 |[Commit detail page](../../repositories/commits.md)<br/>[Pull request detail page](../../repositories/pull-requests.md)<br/>[Email notifications](../../account/emails.md#managing-your-email-notifications)|Variation in percentage points of the coverage value for all files in the commit or pull request|
 |[Pull request detail page](../../repositories/pull-requests.md)|Diff coverage for the changes included in the pull request|
 |[Files page](../../repositories/files.md)|Coverage percentage of each file|
-|[Repository Dashboard](../../repositories/repository-dashboard.md)|Average coverage value for the files in the repository and how the metric is evolving over time|
-|[Codacy badge](../../getting-started/adding-a-codacy-badge.md)|Average coverage value for the files in your repository|
+|[Repository Dashboard](../../repositories/repository-dashboard.md)|Coverage of the most recent commit of the selected branch and its evolution over time|
+|[Codacy badge](../../getting-started/adding-a-codacy-badge.md)|Coverage of the most recent commit of the default branch|
```

Comment:
@fjrdomingues 
Here's [our original discussion](https://codacy.slack.com/archives/C04CMEZT6ER/p1676471176146659?thread_ts=1676469139.213069&cid=C04CMEZT6ER) for context.

I'm also updating the description of the Codacy badge since it appears to show the same value (truncated coverage of the most recent commit). Let me know if this is not the case, thanks!

Reply:
It's actually possible to have coverage badges [for other branches besides the default one](https://docs.codacy.com/getting-started/adding-a-codacy-badge/). So, I'd make the sentence about the badge more generic:

```suggestion
|[Repository Dashboard](../../repositories/repository-dashboard.md)|Coverage of the most recent commit of the selected branch and its evolution over time|
|[Codacy badge](../../getting-started/adding-a-codacy-badge.md)|Coverage of the most recent commit of the configured branch|
```

Reply:
Good catch, thanks!